resolwe_docker_images/qc/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM public.ecr.aws/genialis/resolwebio/common:3.0.1
+
+MAINTAINER Resolwe Bioinformatics authors https://github.com/genialis/resolwe-bio
```

Comment:
Modify your [development environment](https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files) so that the final new line is inserted to the newly created text files. It might be that this is not strictly necessary for Docker files, but it is in general considered a [good practice](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline).